readthedocs/templates/account/signup.html

```diff
@@ -0,0 +1,33 @@
+{% extends "account/base.html" %}
+
+{% load i18n %}
+
+{% block head_title %}{% trans "Signup" %}{% endblock %}
```

Comment:
We are using "Signup" and "Sign up", I think we need to use only one.

Reply:
That's a good catch.

Reply:
Heh, it looks like this is present in allauth itself:
https://github.com/pennersr/django-allauth/blob/2894261/allauth/templates/account/signup.html#L5-L8

Nonetheless, I'll change it for us. I lean toward "sign up" being two words